frontend/src/components/activePage/TableInstance/RowInstanceTitle/RowInstanceTitle.tsx

```diff
@@ -46,8 +52,8 @@ const RowInstanceTitle: FC<IRowInstanceTitleProps> = ({ ...props }) => {
           applyInstanceMutation
         );
         if (result) setTimeout(setEdit, 400, false);
-      } catch {
-        // TODO: properly handle errors
+      } catch (err) {
+        makeErrorCatcher(ErrorTypes.ApolloError)(err as ApolloError);
```

Comment:
It's probably better to make an `apolloErrorCatcher` variable before and use it here, instead of making a new error catcher each time...